src/Service/Sns/tests/Integration/SnsClientTest.php

```diff
@@ -135,8 +135,8 @@ public function testListTopics(): void
 
         $result->resolve();
 
-        // self::assertTODO(expected, $result->getTopics());
-        self::assertSame('change me', $result->getNextToken());
+        self::assertNull($result->getNextToken());
+        self::assertCount(2, $result->getTopics());
```

Comment:
we should also look at the actual topics in the result

Reply:
Is it worth checking out, as the https://github.com/async-aws/aws/blob/b7cbfcbcc7258ad0bef10c7899bf830b8c3a7e2e/src/Service/Sns/tests/Unit/Result/ListTopicsResponseTest.php test already testing the response?

Reply:
Integration tests and unit tests provide a different level of guarantees